training/src/test/scala/com/airbnb/aerosolve/training/FullRankLinearModelTest.scala

```diff
@@ -43,6 +43,11 @@ class FullRankLinearModelTest {
     testFullRankLinear("softmax", 0.8)
   }
 
+  @Test
+  def testFullRankLinearHinge() = {
+    testFullRankLinear("hinge", 0.8)
```

Comment:
did you test higher threshold here? since 'hinge' is doing better, I'd like to see whether it is also improving this simple testing example